lib/cli-update-connection.ts

```diff
@@ -61,26 +69,26 @@ export default function CliUpdateConnection() {
             connectionString: settings.connectionString
         };
 
+        cli.print('Updating connection string...');
         client.datasets.setAllConnections(settings.collection, settings.workspace, settings.dataset, params, (err, result) => {
-            if (err) {
-                cli.error('Error updating connection string');
-                cli.error(err);
-                return;
+            if (err && typeof (callback) === 'function') {
```

Comment:
Stylistic thing, but the parentheses around callback are unecessary.   Imo, it makes typeof look like a function but it's an operator.